en/frames_plane/turbo_timber_evolution.md

```diff
@@ -0,0 +1,203 @@
+- Misc hardware: M3 hardware (standoffs, washers, bolts), XT30 connector, hot glue, heatshrink, Molex Microfit connectors
+- Silicone wiring (14awg for high current, 16awg for low current, 22awg for low power and signals)
+- 3.6Ah 4S LiPo OR 4s2p 18650 LiIon
+
+## Airframe Build
+
+The TTE comes out of the box very complete as-is.
+Servos and linkages have already been installed, and the only real work remaining is finishing work, such as installing the landing gear and horizontal stabilizer.
+For this portion of the assembly, I simply followed the manual.
+
+## FPV Pod
+
+![Window and front fuselage (hatch) with FPV Pod mounted on top](../../assets/airframes/fw/turbo_timber_evolution/fpv_pod_hatch.jpg)
+
+![Underside of hatch showing the FPV pod attachement screws and wires pulled through](../../assets/airframes/fw/turbo_timber_evolution/hatch_underside.jpg)
+
+The FPV pod was mounted on top of the battery hatch using M3 nylon hardware.
+Mounting holes for the pod were located by placing the FPV pod on top (being careful to center it with a ruler) and then punching through the fpv pod mounting holes into the foam with a screwdriver.
+A long M3 nylon screw and a washer on the underside, followed by a washer and standoff on the top of the battery hatch were then be used to mount the FPV pod.
+
+## Pitot Pod
+
+A airspeed sensor is highly recommended for use on fixed wing vehicles.
+This build uses a MS4525DO differential pressure module and pitot tube housed in a 3D printed pod that has a hardpoint hanger for connecting it to the wing.
+
+![Pitot pod/tube sitting on a desk](../../assets/airframes/fw/turbo_timber_evolution/pitotpod1.jpg)
+
+
+Inside the pitot pod, the MS4525DO differential pressure sensor is housed and is connected to the pitot/static tube with a short length of tubing.
+Zipties are used as hoseclamps to prevent the tubing from backing off.
+The i2c and power leads were soldered directly to the MS4525 module, and then hotglue was used to mechanically reinforce the connections.
+
+![Pitot pod opened up showing wires pitot tube connectors and wires soldered and hot-glued in place](../../assets/airframes/fw/turbo_timber_evolution/pitotpod2.jpg)
+
+The pitot/static differential pressure sensor was mounted on the wing (outside the radius of the prop) using a 3D printed "hanger" glued to the leading edge of the wing.
+An M2 screw and nylock hold the pod to the hanger.
+
+![Turbo Timber Evolution Build](../../assets/airframes/fw/turbo_timber_evolution/pitotpodinstalled.png)
+
+These 4 leads were then taped to the underside of the wing running back to the Pixhawk 4 Mini.
+
+The lid of the pitot pod was initially taped in place to allow the setup to be tested and modified if needed after first flight.
+After the first flight, the lid was hot glued in place.
+
+## Flight Computer Installation
+
+
+![Turbo Timber Evolution Build](../../assets/airframes/fw/turbo_timber_evolution/pre_mount_install.jpg)

```

Comment:
I made this image landscape. If you had the plane in front of you, would it be easy to tell which way is the front? If not, should we (you) add an arrow to the image showing the orientation?

Reply:
It should be very easy to tell front vs back because of the servos. I don't think this image is ambiguous if you had the plane sitting in front of you.